admin/includes/template/common/tplHtmlHead.php

```diff
@@ -10,12 +10,19 @@
         <link rel="stylesheet" type="text/css"
               href="<?php echo $entry['href']; ?>" <?php echo(isset($entry['id']) ? 'id="' . $entry['id'] . '"' : ''); ?> <?php echo(isset($entry['media']) ? 'media="' . $entry['media'] . '"' : ''); ?>>
     <?php } ?>
+    <?php if (isset($tplVars['jscriptVars'])) { ?>
+        <script>
+            <?php foreach ($tplVars['jscriptVars'] as $jsName => $jsValue) { ?>
+            var <?php echo $jsName; ?> = '<?php echo $jsValue; ?>'
```

Comment:
probably, although i'm considering reverting this code. The only benefit it currently gives is turning the zcJSFramework file from .php to .js.

There may need to be more support in controllers for this code to be effective .